projects/github-actions/repo-gardening/src/tasks/check-description/index.js

```diff
@@ -507,17 +507,11 @@ The e2e test report can be found [here](https://automattic.github.io/jetpack-e2e
 **Follow this PR Review Process:**
 
 1. Ensure all required checks appearing at the bottom of this PR are passing.
-2. Choose a review path based on your changes:
-    - **A. Team Review:** add the "[Status] Needs Team Review" label
-        - For most changes, including minor cross-team impacts.
-        - Example: Updating a team-specific component or a small change to a shared library.
-    - **B. Crew Review:** add the "[Status] Needs Review" label
-        - For significant changes to core functionality.
-        - Example: Major updates to a shared library or complex features.
-    - **C. Both:** Start with Team, then request Crew
-        - For complex changes or when you need extra confidence.
-        - Example: Refactor affecting multiple systems.
-3. Get at least one approval before merging.
+2. Label this PR with the "[Status] Needs Review" label.
+3. Use [GitHub's Reviewers functionality](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review) to request a review.
+4. Get at least one approval before merging.
+
+ For most changes, e.g updating a team-specific component or a shared library, having a review from your fellow team member should be enough. If this PR makes significant changes to core functionality, or introduces major updates to a shared library or complex features, please reach out to teams or people with expertise. Feel free to ask in the #jetpack-developers channel for help.
 
 Still unsure? Reach out in #jetpack-developers for guidance!`;
```

Comment:
We're mentioning #jetpack-developers twice here. We could probably get rid of one.